utils/staking-miner/src/dry_run.rs

```diff
@@ -63,10 +64,16 @@ async fn print_info<T: EPM::Config>(
 		let snapshot_size =
 			<EPM::Pallet<T>>::snapshot_metadata().expect("snapshot must exist by now; qed.");
 		let deposit = EPM::Pallet::<T>::deposit_for(raw_solution, snapshot_size);
+
+		let score = {
+			let ElectionScore { minimal_stake, sum_stake, sum_stake_squared } = raw_solution.score;
+			[Token::from(minimal_stake), Token::from(sum_stake), Token::from(sum_stake_squared)]
+		};
+
 		log::info!(
 			target: LOG_TARGET,
 			"solution score {:?} / deposit {:?} / length {:?}",
-			&raw_solution.score.iter().map(|x| Token::from(*x)).collect::<Vec<_>>(),
```

Comment:
@kianenigma would be nice if the `iterator API` is public :)

Reply:
unfortunately that iter api is some implementation detail (its order matter). Let's roll with this for now.